### ISSUE TITLE
SCAP Workbench scan issue against RHEL 7.2

### ISSUE THREAD

**Issue by Petaris** [OpenSCAP/scap-security-guide#1055]
System:  Clean minimal install of RHEL 7.2, fully updated
Kernel:  3.10.0-327.10.1.el7.x86_64
Following additional packages installed related to SCAP from the RHEL 7.2 Base repo:  openscap, openscap-scanner, openscap-utils, openscap-python, scap-security-guide
Other utilities:  Confirmed that grep, awk, and sed are all present as well
SCAP Workbench:  Version 1.1.1 running on Windows 7
Profile:  United States Government Configuration Baseline (USGCB / STIG)
Customizations:  Yes, modification to configurable parameters for that baseline

When viewing the scan results items marked as either "Not Scanned" or "Error" have one of the following lines:

Issue 1:  Results listing "None of the check-content-ref elements was resolvable."
Issue 2:  Results that list lines like the following: "/tmp/oscap.GUc5gs/fix-XXrUuujq: line 6: perform_audit_rules_privileged_commands_remediation: command not found"

**Comment by mpreisler** [comment]
Possible related to #590 - remote scanning from Windows.

**Comment by mpreisler** [comment]
```
11:35:40   Petaris | It doesn't seem to list a command that is missing though                                                                                                    │
11:37:08 mpreisler | Petaris: OK, one more thing I want you to try. Please download                                                                                              │
                   | https://github.com/OpenSCAP/scap-security-guide/releases/download/v0.1.28/scap-security-guide-0.1.28.zip and open ssg-rhel7-ds.xml from that with SCAP      │
                   | Workbench                                                                                                                                                   │
11:37:25 mpreisler | there is a chance that we fixed this and just haven't released new workbench with it bundled                                                                │
11:37:44   Petaris | ok, let me try that                                                                                                                                         │
11:37:51 mpreisler | ah, I know exactly what the problem is                                                                                                                      │
11:38:00 mpreisler | we already have a bug for it :)                                                                                                                             │
11:38:31 mpreisler | https://github.com/OpenSCAP/scap-security-guide/issues/590                                                                                                  │
11:38:36 mpreisler | unfortunately we have bee postponing it for a while                                                                                                         │
11:39:12 mpreisler | the reason why you get command not found is a version mismatch between scap-security-guide installed on the scanned machine and the ssg version bundled     │
                   | with workbench                                                                                                                                              │
11:39:38   Petaris | ahh, ok                                                                                                                                                     │
11:40:04   Petaris | Version     : 0.1.25                                                                                                                                        │
11:40:04   Petaris | Release     : 3.el7                                                                                                                                         │
11:40:08   Petaris | That is on the system                                                                                                                                       │
11:40:38 mpreisler | yes, scap workbench probably has a newer version bundled and the remediations try to use the shared remediation functions from an older version             │
11:40:49 mpreisler | Petaris: Can I paste this conversation into the bug?                                                                                                        │
11:40:53   Petaris | Sure
```

**Comment by mpreisler** [comment]
This has been fixed with SSG PR #1270
